src/components/ConnectionsHeader/icons.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Octicons from 'react-native-vector-icons/Octicons';
+import Entypo from 'react-native-vector-icons/Entypo';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+import AntDesign from 'react-native-vector-icons/AntDesign';
+
+export const ALARMS = {
+  'Circuit Integrity Failed': {
+    status: 'Circuit Integrity Passing',
+    icon: (status: boolean) => (
+      <Octicons
```

Comment:
We are using numerous types of icons - is it not possible somehow to use the same icon set? I guess it's difficult to find all applicable icons

Reply:
Unfortunately no,  I also had this in mind but sadly each icons library doesnt have all the icons and there was not a way to default import them all.

Reply:
Okay, no worries, that sounds good